os-datavirt/added/standalone-openshift.xml

```diff
@@ -525,14 +525,14 @@
             <preparedplan-cache infinispan-container="teiid-cache"/>
             <transport name="embedded"/>
             <transport name="odata">
-                <authentication security-domain="teiid-security"/>
+                <authentication security-domain="##ODATA_SECURITY_DOMAIN##"/>
```

Comment:
@bdecoste, is it required having all transports configured? What if the customer want to use only one of them?